sdk/go/request.go

```diff
@@ -45,57 +45,53 @@ func (r OnlineFeaturesRequest) buildRequest() (*serving.GetOnlineFeaturesRequest
 		}
 	}
 	return &serving.GetOnlineFeaturesRequest{
-		Features:   features,
+		Features:   featureRefs,
 		EntityRows: entityRows,
 	}, nil
 }
 
-// buildFeatures create a slice of FeatureReferences from a slice of "<project>/<feature_name>:<feature-set-version>"
-// It returns an error when the format is invalid
-func buildFeatures(featureReferences []string, defaultProject string) ([]*serving.FeatureReference, error) {
-	var features []*serving.FeatureReference
-
-	for _, featureRef := range featureReferences {
-		var project string
-		var name string
-		var version int
-		var featureSplit []string
-
-		projectSplit := strings.Split(featureRef, "/")
-
-		if len(projectSplit) == 2 {
-			project = projectSplit[0]
-			featureSplit = strings.Split(projectSplit[1], ":")
-		} else if len(projectSplit) == 1 {
-			project = defaultProject
-			featureSplit = strings.Split(projectSplit[0], ":")
-		} else {
-			return nil, fmt.Errorf(ErrInvalidFeatureName, featureRef)
+// Creates a slice of FeatureReferences from string representation in
+// the format featureset:feature.
+// featureRefStrs - string feature references to parse.
+// project - Sets the project in parsed FeatureReferences. Otherwise ""
```

Comment:
Think you have a type there with `""`

Reply:
Not a typo. Its an optional parameter which users should pass `""` if they project is not specified as golang does not support default parameters.